web/client/components/catalog/editor/AdvancedSettings/RasterAdvancedSettings.js

```diff
@@ -158,5 +159,6 @@ export default ({
         {!isNil(service.type) && service.type === "csw" &&
         <CSWFilters filter={service?.filter} onChangeServiceProperty={onChangeServiceProperty}/>
         }
+        {(!isNil(service.type) ? service.type === "wms" : false) && (<WMSDomainAliases service={service} onChangeServiceProperty={onChangeServiceProperty} />)}
```

Comment:
we usually use ternary like this 
```suggestion
        {(!isNil(service.type) && service.type === "wms" ? (<WMSDomainAliases service={service} onChangeServiceProperty={onChangeServiceProperty} />) : null}
```

Reply:
Thanks, simplified it to "condition && jsx"